cmd/minishift/cmd/start.go

```diff
@@ -907,3 +912,8 @@ func startTray() error {
 	}
 	return nil
 }
+
+func updateConsoleURL(output string) string {
+	re := regexp.MustCompile(":8443")
+	return re.ReplaceAllLiteralString(output, ":8443/console")
```

Comment:
I am planning to put a better regular expression here.

Reply:
? so this is WIP?

Reply:
I tried to put a regular expression to match the IP:8443. But that looks complicated to do this simple thing. Lets go ahead with this one.